### ISSUE TITLE
US Bitcoin Reserve

### ISSUE THREAD

**Issue by jonmarrs** [responsible-financial-innovation-act22/RFIA-bill#139]
The United States has a gold reserve. The US stores gold bullion in secure locations such as Fort Knox. The United States also has a Strategic Petroleum Reserve. The United States should also have a US Bitcoin Reserve. The US should start stockpiling Bitcoin just as it stockpiles gold and oil. Gold was once the foundation of money, as the US dollar was backed by gold, and the dollar could be converted to gold. The gold-dollar peg was broken by President Nixon in 1971, yet the US still maintains its gold reserve as a storehold of national wealth. Looking towards the future, as Bitcoin plays an increasingly important role in the global economy and international trade as a decentralized and politically neutral currency, it seems likely that Bitcoin will mature into a storehold of wealth for nation states, similar to gold. Bitcoin might even become the next global reserve currency. Therefore, it seems prudent for the United States to start accumulating and HODLing Bitcoin. This sovereign Bitcoin HODLing strategy is already being implemented in countries such as El Salvador. Furthermore, it is crucial that the United States starts to stockpile significant amounts of Bitcoin long before our rivals such as Russia and China start to accumulate Bitcoin. Whichever country stockpiles the most Bitcoin first shall inherit the earth and ensure its economic prosperity and financial dominance as the next global superpower. The US needs a Bitcoin Fort Knox or US Bitcoin Reserve.

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175763456-4df61c1a-bd14-460a-adf0-81f87cd476eb.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175763487-e2bb74c2-37cf-42c6-8441-0bb7f3b53d21.png)

**Comment by jonmarrs** [comment]
"As an amusing thought experiment, imagine that Bitcoin is successful and
becomes the dominant payment system in use throughout the world.  Then the
total value of the currency should be equal to the total value of all
the wealth in the world. Current estimates of total worldwide household
wealth that I have found range from $100 trillion to $300 trillion. With
20 million coins, that gives each coin a value of about $10 million.

So the possibility of generating coins today with a few cents of compute
time may be quite a good bet, with a payoff of something like 100 million
to 1! Even if the odds of Bitcoin succeeding to this degree are slim,
are they really 100 million to one against? Something to think about..."

-Hal Finney
[Re: Bitcoin v0.1 released](https://www.mail-archive.com/search?l=cryptography@metzdowd.com&q=subject:%22Re%5C%3A+Bitcoin+v0.1+released%22&o=newest)](https://www.mail-archive.com/cryptography@metzdowd.com/msg10152.html)

**Comment by jonmarrs** [comment]
[The Bullish Case for Bitcoin](https://vijayboyapati.medium.com/the-bullish-case-for-bitcoin-6ecc8bdecc1)

**Comment by jonmarrs** [comment]
The United States currently has ~$30 trillion of national debt. As an amusing thought experiment, imagine that the United States accumulates ~$300 billion of Bitcoin. As Bitcoin matures into a global reserve currency/asset, the Bitcoin market capitalization could grow 100x from <$1 trillion to >$100 trillion, allowing the United States to completely pay off its national debt simply by HODLing ~$300 billion of Bitcoin (a fraction of the amount of money that is spent on defense, a fraction of the amount of money that has been printed since 2019, and a fraction of the Federal Reserve balance sheet).

**Comment by pivotman319-owo** [comment]
> The US needs a Bitcoin Fort Knox or US Bitcoin Reserve.

How about no?

**Comment by HamishSinclair** [comment]
I-I just can't even dude. When will you trumpcels finally realize how much of a heckin' yikes crypto is?

**Comment by twl-surrogate** [comment]
its over for trumpcels

**Comment by amcgregor** [comment]
Intriguing chart to wield. Bitcoin, on…

Verifiable: only after substantial delay. Charge-backs are a thing, given you have to wait for the block to get signed before the transaction is irreversible.

Fungible: uh, okay? One floating point number is as good as another.

Portable: if moving the physical wallet DB around, sure. Also easy to steal, because of this. Less portable if actually utilizing the inherent process of transfer, again, chargebacks.

Durable: still have your Geocities, Tucows, or Angelfire homepage from the 90's? Didn't think so.

Divisible: again, chopping up floating point numbers is mostly trivial. Fun failure, though, is that floating point numbers are inaccurate by design. Add `0.1` to itself `10` times. You won't get `1.0`.

<img width="404" alt="Screen Shot 2022-06-25 at 10 52 05" src="https://user-images.githubusercontent.com/76750/175778908-0f87ed40-de21-41d4-8ecd-0e9b455c0706.png">

I'm fairly confident this isn't how math works.

Scarce: artificially so, with lagged moderation controls that should theoretically adapt to technological innovation, at the cost of the immediate obsolescence of prior techniques as new arise. (Ref: CPU → GPU, GPU → FPGA, …)

Censorship resistant: let me introduce you to the [Great Firewall of China](https://en.wikipedia.org/wiki/Great_Firewall). This is a leading contributor to the transaction throughput and latency issues, as transactions end up round-tripped through China twice.

"Unforgeable Costliness" → congratulations, you've found a lament box. A puzzle that if solved, destroys you. The cost is the habitability of our planet through proactive destruction by way of unlimited resource expenditure. For what purpose?

To say "I agree that X said Y to Z."

Proof-of-work blockchain.
It's how you end the world while getting to think it's an awfully cool way to do it.

**Comment by hallambaker** [comment]
> The United States has a gold reserve. The US stores gold bullion in secure locations such as Fort Knox. The United States also has a Strategic Petroleum Reserve. The United States should also have a US Bitcoin Reserve. 

My, the creation of a new meme!

_The United States has a gold reserve. The US stores gold bullion in secure locations such as Fort Knox. The United States also has a Strategic Petroleum Reserve. The United States should also have a ____ reserve._

As in.

The United States has a gold reserve. The US stores gold bullion in secure locations such as Fort Knox. The United States also has a Strategic Petroleum Reserve. The United States should also have a Beanie Baby reserve.

**Comment by CrazedCoding** [comment]
> 

TOTALLY! Let's end the world like accountable altruist so we can go to heaven! Trump 2024!

**Comment by CrazedCoding** [comment]
> The United States currently has ~$30 trillion of national debt. As an amusing thought experiment, imagine that the United States accumulates ~$300 billion of Bitcoin. As Bitcoin matures into a global reserve currency/asset, the Bitcoin market capitalization could grow 100x from <$1 trillion to >$100 trillion, allowing the United States to completely pay off its national debt simply by HODLing ~$300 billion of Bitcoin (a fraction of the amount of money that is spent on defense, a fraction of the amount of money that has been printed since 2019, and a fraction of the Federal Reserve balance sheet).

Pay debts!? Who needs balance when you've got Jesus!

**Comment by jonmarrs** [comment]
> Intriguing chart to wield. Also links; [here's one pointing out its failures](https://blog.plan99.net/the-resolution-of-the-bitcoin-experiment-dabb30201f7). (I have greater trust in articles not trying to sell me things, generally.)
> 
> **Verifiable:** only after substantial delay. Charge-backs are a thing, given you have to wait for the block to get signed before the transaction is irreversible.
> 
> **Fungible:** uh, okay? One floating point number is as good as another.
> 
> **Portable:** if moving the physical wallet DB around, sure. Also easy to steal, because of this. Less portable if actually utilizing the inherent process of transfer, again, chargebacks.
> 
> **Durable:** still have your Geocities, Tucows, or Angelfire homepage from the 90's? Didn't think so.
> 
> **Divisible:** again, chopping up floating point numbers is mostly trivial. Fun failure, though, is that floating point numbers are inaccurate by design. Add `0.1` to itself `10` times. You won't get `1.0`.
> 
> <img alt="Screen Shot 2022-06-25 at 10 52 05" width="404" src="https://user-images.githubusercontent.com/76750/175778908-0f87ed40-de21-41d4-8ecd-0e9b455c0706.png">
> 
> I'm fairly confident this isn't how math works.
> 
> **Scarce:** artificially so, with lagged moderation controls that should theoretically adapt to technological innovation, at the cost of the immediate obsolescence of prior techniques as new arise. (Ref: CPU → GPU, GPU → FPGA, …) (And many aren't designed to be scarce. Doge is literally a joke because of this particular point.)
> 
> **Censorship resistant:** let me introduce you to the [Great Firewall of China](https://en.wikipedia.org/wiki/Great_Firewall). This is a leading contributor to the transaction throughput and latency issues, as transactions end up round-tripped through China twice.
> 
> **"Unforgeable Costliness"** → congratulations, you've found a lament box. A puzzle that if solved, destroys you. The cost is the habitability of our planet through proactive destruction by way of unlimited resource expenditure. For what purpose?
> 
> To say "I agree that X said Y to Z."
> 
> Proof-of-work blockchain. It's how you end the world while getting to think it's an awfully cool way to do it.

Verifiable: Bitcoin transactions can instantly be verified by checking by checking the blockchain, a public ledger. You can run your own node and have access to your own copy of the blockchain, which means you don't have to trust a bank or payment processor, you can verify the authenticity of the transaction yourself. Bitcoin transactions post faster (<1 second) than credit/debit card transactions (~1 minute). Bitcoin transactions also settle faster (<10 minutes) that credit/debit card transactions (days). Bitcoin transactions can also be even faster and cheaper if you use Layer 2 / Lightning Network instead of Layer 1 / blockchain. We accept debit/credit card fraud as a given, as transactions cannot be verified until days later. Credit/debt card payment processors such as Visa/AmEx/MasterCard charge 2%-3% transaction fees to cover this fraud. With Bitcoin (especially with the Lightning Network), the transaction times are much faster, the fees are much lower, and transactions can be verified within 10 minutes. With Bitcoin, fraud is prevented since your private keys are never revealed during the transaction, preventing theft of the means to sign/authorize transactions.

Fungible: Correct, computer digits are fungible. Bitcoin is pristine and fungible. Fiat paper bills can get ripped/torn/soiled and have serial numbers, making each dollar bill different.

Portable: You can carry around a Bitcoin wallet on your smartphone. Your wallet can be locked with a pin code and/or biometric/finger/face scan. You can also carry around a separate Bitcoin hardware wallet if you want. Fundamentally, Bitcoin is just information, so you can walk around with bitcoin if you have 12-24 words (seed phrase) written down on a piece of paper, or memorized in your head. There is nothing more portable than Bitcoin, as the information can be transcribed into any medium that you wish. This is in stark contrast to carrying around gold bullion/coins, fiat paper bills, and fiat coins. Also, there are restrictions on accessing your bank account and credit/debit card when you are traveling, and banks also have withdrawal limits.

Durable: Bitcoin is extremely durable due to the decentralized nature of the blockchain / distributed ledger. Even if your house burns down, there are >10,000 copies of your account details and Bitcoin balanced stored on Bitcoin nodes dispersed around the world, and even on satellites in space. It is much more likely that your bank's computers could get destroyed and your checking acount / savings account / brokerage account / retirement account balance will get wiped out, but your Bitcoin balance will be eternal. As long as you have access to your 12-24 words, you can always recover your Bitcoin savings. That is extreme durability.

Divisible: It is trivial to divide Bitcoin into smaller and smaller amounts, making Bitcoin useful for microtransactions. If you want to pay someone less than $0.01 you are going to have a lot of trouble. The problem gets even worse if you try to divide gold. Good luck paying for a cup of coffee with gold bullion/coins... maybe you can shave off some gold dust with a file and weigh out a few micrograms with a well-calibrated scale.

Scarce: The Bitcoin supply will never be more than 21 million. The supply of fiat currency is potentially infinite; it depends on a few arbitraty mouseclicks or keystrokes from Jerome Powell. The supply of gold is unkown, but there is plenty of gold in the universe. Gold supply inflates at about 2% per year, so it is more scarce than fiat, but occassionaly there are new deposits discovered such as the recent gold motherlode found in Uganda. I agree that most cryptocurrencies aren't scarce, especially because most cryptocurrencies aren't decentralized. However, Bitcoin is truly decentralized, and so the currency supply will never be increased. Bitcoin is sound money.

Censorship Resistant: Bitcoin can be accessed through multiple different networks and communication protocols. The internet is the most common way of accessing the Bitcoin network, but you could also access it via satellite uplink. You could also access Bitcoin via phone line, morse code, CD ROM, USB stick, pen and paper. Bitcoin is just information, so if you can send/receive information in some way you can write/read to/from the Bitcoin blockchain.


Unforgeable Costliness:  This is an important property to ensure that the monetary system can't be gamed and manipulated. If there is no cost for producing money, than elites will find it trivial to create new money at the expense of everyone else as well as the environment. Consider all of the inflation that has been created by the Federal Reserve printing trillions of dollars out of thin air. You are now paying very high gas prices partly as a result of this manipulation of money. Wealth inequality is increasing as the Fed prints money and gives it away to hedge funds, banks, and wealthy elites by buying artificially inflated Treasury bonds and financial assets. The Fed artificially lowers interest rates, which leads to more borrowing and money in the system, and makes the rich richer by artificially increasing the value of financial assets, while making the poor poorer by artificially increasing inflation. The artificially low interest rates also leads to asset bubbles, an increase in speculation and gambling, and malinvestments; which eventually leads to a market crash and recession/depression like we are seeing now. Bitcoin fixes this by allowing the free market and sound money to function instead of interfering with the market and monetary policy. Unforgeable costliness is the reason why gold was selected as money for thousands of years after humanity had experimented with countless other forms of money (cowrie shells, beads, tobacco, salt, livestock, etc.)

Proof-of-Work saves the world by allowing humanity to trust math, code, and physics/thermodynamics; rather than having to trust elites/banks/governments/institutions/kings/tyrants. Absolute power corrupts absolutely. Bitcoin allows us to stop trusting humans who always break their promises and devalue the money supply (FDR changing the dollar-gold peg in 1933, Nixon completely removing the dollar-gold peg in 1971) and instead rely on immutable code in a decentralized network that can never be altered. The benefits of an incorruptible and apolitical sound money are profound for the benefit of humanity. Bitcoin gets rid of manipulation/corruption/noise/errors/lies and allows humanity to collaborate with eachother with pure signal/integrity/truth. Money is information, and Bitcoin is the purest form of monetary information, allowing free markets to function as they should. You can either have a limited currency supply with abundant and cheap goods/services (Bitcoin), or you can have an abundant currency suppy with limited and expensive goods/services (fiat).

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175790225-8d6e18ce-0b9e-4f6a-b55a-ad308b5fd0a6.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175790249-2b016c7c-494f-4279-8f43-437fffde9b9f.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175790321-211f3551-07a0-4dfa-9a46-8458ecffb4bc.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175790466-aef0548a-20e0-4488-9cb1-1db41abae013.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175790515-526d0509-b022-4989-85e4-1e439cb7e6b5.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175790565-76f644c0-5b26-4ac1-bbd6-f80170b2bb71.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175790693-3a94dc42-e9c5-41e5-8c9c-00aab04eaa95.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175790725-1bb1162a-5f67-4a37-927b-11d6d0a2fb30.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175790741-3462380b-fc5a-4558-abf2-c5abb243e090.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175790801-72201273-e765-4220-a8c6-f1f925fbbe9a.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175790843-94d33a14-7637-464c-9e97-814eea3f2d0a.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175790927-55a00307-8ed8-4c96-a8b7-fcdf0f24c42e.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175790969-b1656aa8-d7e6-49ab-989b-8a8f7872770e.png)

**Comment by jonmarrs** [comment]
[Bitcoin is Key to an Abundant, Clean Energy Future](https://assets.ctfassets.net/2d5q1td6cyxq/5mRjc9X5LTXFFihIlTt7QK/e7bcba47217b60423a01a357e036105e/BCEI_White_Paper.pdf)

**Comment by jonmarrs** [comment]
[This Machine Greens](https://www.youtube.com/watch?v=b-7dMVcVWgc)

**Comment by amcgregor** [comment]
@jonmarrs I, too, could violate ToS and spam the ever living ████ out of a thread, but I'm not actually that stupid. Screaming the loudest a valid argument does not make.

**Comment by amcgregor** [comment]
> **Verifiable**: Bitcoin transactions can instantly be verified by checking the blockchain, a public ledger.

[For various definitions of "instant".](https://bitinfocharts.com/comparison/bitcoin-confirmationtime.html#3y) July 2021 peaked at ~25 minutes, average hovers around 10 minutes. Admittedly, this is better than the historical averages I remember when first looking into the technology.

This does pose a chargeback risk, with a fairly substantial window.

> You can run your own node and have access to your own copy of the blockchain, which means you don't have to trust a bank or payment processor, you can verify the authenticity of the transaction yourself.

Yes, and whoever controls a dominant proportion of the network power writes history. The "whom" may be a group, with notable demonstrations of such a dominant group sitting at a single table on multiple occasions. (Which poses its own inherent risks, entirely separate from the possibility of collusion.)

The ever-growing nature of the chain, however, imposes restrictions on the devices capable of such a feat. This encourages movement for mobile devices back to centralized silos of authority.

> Bitcoin transactions post faster (<1 second) than credit/debit card transactions (~1 minute).

False. See the link given above, or [this one from December 2017](https://coincentral.com/how-long-do-bitcoin-transfers-take/).

> Bitcoin transactions also settle faster (<10 minutes) than credit/debit card transactions (days).

Consider that this [could potentially be an intentional consumer protection measure](https://surecard.in/learn/credit-card-chargeback-fraud-protection/).

> Bitcoin transactions can also be even faster and cheaper if you use Layer 2 / Lightning Network instead of Layer 1 / blockchain.

> We accept debit/credit card fraud as a given, as credit/debit card transactions cannot be verified until days later, and the authorization details are trivial to steal while cards are easily lost/stolen.

I… haven't used a physical card in more than 10 years. But, hey, that's me.

> Credit/debit card payment processors such as … charge a 2%-3% transaction fee on every transaction to cover this fraud and to generate excessive profit with their payment processing cartel/monopoly.

You may have missed the industry disruption of Square, then. 🤔

> …fraud is prevented since your private keys are never revealed during the transaction, preventing theft of the means to sign/authorize transactions.

You're too hyper-focused on specific technical solutions, and not generalized fraud which covers far more than having your wallet stolen.

> **Fungible**: Correct, computer digits are fungible. Bitcoin is pristine and fungible. Fiat paper bills can get ripped/torn/soiled and have serial numbers, making each dollar bill different.

Every transaction on a blockchain is perfectly traceable to its moment of origin, leading to a network of complete transaction transparency if you can at all correlate wallet identifiers to individuals. It's shockingly difficult to be actually anonymous.

> **Portable**: … There is nothing more portable than Bitcoin…

😂 Sorry, I don't think that requires much more than an emoji response.

> **Durable**: …there are >10,000 copies of your account details…

But not the keys you require to utilize any of them.

> It is much more likely that your bank's computers could get destroyed and your checking acount / savings account / brokerage account / retirement account balance will get wiped out…

There are regulations regarding IT + SecOps standards, certifications and audits, as well as government-backed insurance (at least, up here in Canada) on account balances in case of catastrophe. You clearly do not know what you are speaking of.

> …your Bitcoin balance will be eternal.

Still use Kazaa? Have that Geocities/Tucows/Angelfire homepage from the 90's? Haven't spun up my Hotline client in a while; last time I did, there were no servers I could find, alas.

> Your Bitcoin accounts will survive a nuclear attack…

Let me stop you there. It won't matter, because nothing will. Your blind confidence of your own survival, and the survival of a technological age in such a situation is sadly evident.

> **Divisible**: It is trivial to divide Bitcoin into smaller and smaller amounts, making Bitcoin useful for microtransactions. [argumentum-ad-absurdum digression redacted]

I already demonstrated a fun example of where that goes wrong, with computers.

> **Scarce**: The supply of fiat currency is potentially infinite; it depends on a few arbitrary mouseclicks or keystrokes…

Yup-nope.

> Bitcoin is truly decentralized…

[Oh really?](https://www.bitcoinmining.com/chinas-dominance-in-bitcoin-mining/) [Are you sure about that?](https://www.coindesk.com/markets/2021/10/13/bitcoin-mining-after-the-china-ban-us-dominance-is-set-to-continue/) It's just more obfuscated.

> **Censorship Resistant**: Bitcoin can be accessed through multiple different networks and communication protocols. The internet is the most common way of accessing the Bitcoin network, but you could also access it via satellite uplink [internet]. You could also access Bitcoin via phone line [internet], [morse code, CD ROM, USB stick, pen and paper](https://en.wikipedia.org/wiki/IP_over_Avian_Carriers) [internet]. Bitcoin is just information, so if you can send/receive information in some way you can write/read to/from the Bitcoin blockchain.

False. You also require the computational power to process the information, which is restrictive. _Good **luck**_ performing AES-256 and SHA-512 by pen and paper.

> **Unforgeable Costliness**: This is an important property to ensure that the monetary system can't be gamed and manipulated.

And intentionally destroys the planet. 👍
Paranoia skipped. Yikes, that was a lot on that one. 😕

> Proof-of-Work saves the world by allowing humanity to trust math, code, and physics/thermodynamics…

And intentionally destroys the planet. 👍 (May be getting recursive, here…)

> The benefits of an incorruptible and apolitical sound money are profound for the benefit of humanity.

Only if the planet remains habitable, and if no country wields network dominance—so you're trusting the Chinese and American governments, on this one, with China already demonstrating a willingness to exert total control over information content within their borders.

It's also not "money" in any meaningful sense. It's a speculative asset store and tradable "future". Buy a "coin", you're making a bet against your future capacity to sell it to someone else who wants one for roughly the same amount of equivalent real money usable to purchase everything else in daily life.

Apologies for the separated response; yours got somewhat buried, there.

**Comment by jonmarrs** [comment]
> @jonmarrs I, too, could violate ToS and spam the ever living ████ out of a thread, but I'm not actually that stupid. Screaming the loudest a valid argument does not make.
> 
> 
> 
> [A](https://www.thebalance.com/how-much-power-does-the-bitcoin-network-use-391280) [B](https://hbr.org/2021/05/how-much-energy-does-bitcoin-actually-consume) [C](https://digiconomist.net/bitcoin-energy-consumption) (single-transaction equivalent to 1.8 million VISA transactions, or a hair shy of _50 days of average household electrical consumption_) [D](https://www.forbes.com/advisor/investing/cryptocurrency/bitcoins-energy-usage-explained/) (Forbes article)
> 
> 
> 
> Edit to provide concise links and question: so, what's your take? Into which orifice are you accepting contributions? [And, of course, Yet Another Fork™. What happened to Bitcoin XT? Oh, right; politics and drama.]

Layer 1 (blockchain) is Bitcoin's settlement layer, and settlement can happen every 10 minutes (every time a new block of transactions is added to the chain). Layer 2 ([Lightning Network](https://youtu.be/rrr_zPmEiME)) is Bitcoin's transaction/payment layer, for posting transactions. Transactions in the traditional financial system take [days to settle](https://www.actumprocessing.com/ach-settlement-times/). Payment processors like debit/ credit/ Visa/ Mastercard/ American Express/ Discover/ Venmo/ PayPal in the traditional financial system can post transactions in ~1 minute (compared to Lightning Network's fraction of a second), but charge a large [~3% fee](https://www.fool.com/the-ascent/research/average-credit-card-processing-fees-costs-america/) to the vendor which passes on the cost to the customer via higher prices (compared to a much smaller [~$0.01 fee](https://markets.businessinsider.com/news/currencies/the-lightning-network-is-driving-mainstream-bitcoin-adoption-2021-8) paid by the customer with Lightning Network). Bitcoin and Lightning Network is much better for consumers, although banks and payment processors will not make as much money, because they will no longer be able to charge consumers and businesses such outrageous fees since they will finally have some free-market competition from a free and open source financial network like the internet of a permission-based closed source bank-controlled payment network cartel/monopoly.

Regarding Bitcoin's energy use, I think the amount of energy that Bitcoin uses is worth every Watt, especially when you consider that Bitcoin uses a minuscule amount of global energy, and electric clothes dryers and Christmas lights each use more energy than Bitcoin does. You also have to consider all of the energy wasted by armored Brinks trucks driving around to collect cash from retail stores, all the ATM machines, all the brick-and-mortar bank branches, and the energy-intensive infrastructure of the traditional financial system. You also have to consider the great financial and energy expense of the US military industrial complex to enforce consensus of the US dollar as the global reserve currency, and the fact that the [petrodollar system](https://bitcoinmagazine.com/culture/the-hidden-costs-of-the-petrodollar) is underpinned by massive oil production and burning of fossil fuels.

Nobody is paying me anything. I just want a financial system that works for all of us instead of just the top 1%. Fiat currency creates massive wealth inequality and socioeconomic issues such as poverty, homelessness, unaffordable housing, inflation, artificial financial asset appreciation for the wealthy, and destruction of wealth for savers and the working class who live paycheck-to-paycheck. With fiat currency and the traditional banking system, wealthy elites who have access to the fiat money printer (e.g., hedge funds, Wall Street, corporations, billionaires) have an unfair advantage relative to everyone else as they can sell their assets/Treasuries/bonds to the Fed at inflated prices and they can borrow money at ~0% interest and buy more financial assets with the newly printed money. This is known as the [Cantillon Effect](https://whatismoney.info/cantillon-effect-bitcoin/). For example, BlackRock buys up all the single family homes to protect themselves from the Fed-created inflation, driving up real estate prices and making housing unaffordable for the average person. Bitcoin fixes this by leveling the playing field and ensuring that everyone follows the same set of rules. Bitcoin does this by decentralizing the monetary system getting and fixing the money supply at 21 million coins. This gets rid of the fiat money printer so that nobody has special privileges or power over anyone else. With Bitcoin, there are no central bankers to rig the game and steal from the poor and give to the rich via printing money and inflation.

**Comment by jonmarrs** [comment]
> And intentionally destroys the planet. 👍 Paranoia skipped. Yikes, that was a lot on that one. 😕
> 
> > Proof-of-Work saves the world by allowing humanity to trust math, code, and physics/thermodynamics…
> 
> And intentionally destroys the planet. 👍 (May be getting recursive, here…)
> 
> > The benefits of an incorruptible and apolitical sound money are profound for the benefit of humanity.
> 
> Only if the planet remains habitable, and if no country wields network dominance—so you're trusting the Chinese and American governments, on this one, with China already demonstrating a willingness to exert total control over information content within their borders.

Bitcoin only uses [~0.1% of global energy](https://www.lynalden.com/bitcoin-energy/)... how is that destroying the planet?

More importantly: How is possible to solve climate change from an economic system that REQUIRES inflation?

Instead of embracing the exponential technological gains that naturally bring deflation and broader based abundance, fiat monetary policy (globally) is fighting it - keeping society on a treadmill of ever higher prices and needing more consumption and more production....forever.

The real "inconvenient truth" is that climate cannot be solved from within a system that requires perpetual growth (and will manipulate money to get it).

Because ALL of the innovation (and lower prices) that entrepreneurs bring to the market MUST be offset with more manipulation of money to keep the inflationary economic system running.

The only way out (for our climate and humanity) is from a different system -- one that allows for deflation.

That alternative deflationary financial system is Bitcoin.

I realize the implications of a transition between two systems that operate differently. It doesn't change the facts. 

Ignoring those facts will certainly be more painful (for our climate and everything else).

Inflation = climate change. 

The global inflationary monetary system is the cause of climate change.

There is no possible way to save this planet through a system that must grow “forever” by manipulating money. 

I realize that seems counterintuitive and hard to understand from a system that teaches you differently....but it is true. 

Connect the dots! A system problem cannot be solved through the system that creates it. 

Bitcoin saves the planet.

Bitcoin fixes this.
![image](https://user-images.githubusercontent.com/40682667/175833147-c24242b9-7a41-47c7-8a82-02068a79e754.png)

**Comment by WCWedin** [comment]
Capitalism, not inflation, is what's destroying the planet, and the only beneficiaries of deflation are those who already hold substantial assets.

**Comment by jonmarrs** [comment]
> Capitalism, not inflation, is what's destroying the planet, and the only beneficiaries of deflation are those who already hold substantial assets.

You have it backwards. Inflation is what is killing the planet. Inflation = waste and misallocated resources. With inflation, people are incentivized to go into debt and over-consume. Inflation increases the time preference of individuals; people want to consume today instead of saving for tomorrow. With inflation, consequences are deferred until later.
